docs/azure/sdk/includes/enable-managed-identity-azure-cli.md

```diff
@@ -1,5 +1,5 @@
 ---
-ms.custom: devx-track-azurecli
+ms.custom:
```

Comment:
This should either have a value, or be entirely removed. An empty value isn't valid, right?

Reply:
@IEvangelist , ahhh, got it!  I'll close this PR and open a new one.  Thanks for the additional explanation.  Makes sense.